src/emulator/functionsEmulator.ts

```diff
@@ -954,6 +960,7 @@ export class FunctionsEmulator implements EmulatorInstance {
     envs.TZ = "UTC"; // Fixes https://github.com/firebase/firebase-tools/issues/2253
     envs.FIREBASE_DEBUG_MODE = "true";
     envs.FIREBASE_DEBUG_FEATURES = JSON.stringify({ skipTokenVerification: true });
+    // TODO(danielylee): Support timeouts. Temporarily dropping the feature until we finish refactoring.
```

Comment:
Does this put a freeze on CLI releases? Should this be checked into a feature branch?

Reply:
Technically yes, though timeouts are never really enforced.

What we lose is a warning message that would've let users know that the timeout has been reached and in prod would've killed the fn.

I'm wasn't too worried about loosing this capability, but that's kinda slacking. I'll bring this up for discussion offline.